datafusion/sqllogictest/src/test_context.rs

```diff
@@ -361,3 +369,893 @@ fn create_example_udf() -> ScalarUDF {
         adder,
     )
 }
+
+fn register_union_tables(ctx: &SessionContext) {
```

Comment:
Can't we create table in slt file?

Reply:
Until #10206 is fixed, I don't think so. Anyway, with the core of this PR moved to arrow-rs, all these tests are unit tests on arrow-rs now :pray: